samples/snippets/src/main/java/com/example/spanner/admin/generated/CreateDatabaseWithDefaultLeaderSample.java

```diff
@@ -0,0 +1,81 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.admin.generated;
+
+//[START spanner_create_database_with_default_leader]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.spanner.SpannerException;
+import com.google.cloud.spanner.SpannerExceptionFactory;
+import com.google.cloud.spanner.admin.database.v1.DatabaseAdminClient;
+import com.google.common.collect.ImmutableList;
+import com.google.spanner.admin.database.v1.CreateDatabaseMetadata;
+import com.google.spanner.admin.database.v1.CreateDatabaseRequest;
+import com.google.spanner.admin.database.v1.Database;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class CreateDatabaseWithDefaultLeaderSample {
+
+  static void createDatabaseWithDefaultLeader() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    final String instanceName = "projects/my-project/instances/my-instance-id";
+    final String databaseId = "my-database-name";
+    final String defaultLeader = "my-default-leader";
+    createDatabaseWithDefaultLeader(instanceName, databaseId, defaultLeader);
+  }
+
+  static void createDatabaseWithDefaultLeader(String instanceName, String databaseId,
+      String defaultLeader) throws IOException {
+    DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.create();
+
+    try {
+      OperationFuture<Database, CreateDatabaseMetadata> op1 =
+          databaseAdminClient.createDatabaseAsync(
+              CreateDatabaseRequest.newBuilder()
+                  .setParent(instanceName)
+                  .setCreateStatement("CREATE DATABASE `" + databaseId + "`")
+                  .addAllExtraStatements(
+                      ImmutableList.of("CREATE TABLE Singers ("
+                              + "  SingerId   INT64 NOT NULL,"
+                              + "  FirstName  STRING(1024),"
+                              + "  LastName   STRING(1024),"
+                              + "  SingerInfo BYTES(MAX)"
+                              + ") PRIMARY KEY (SingerId)",
+                          "CREATE TABLE Albums ("
+                              + "  SingerId     INT64 NOT NULL,"
+                              + "  AlbumId      INT64 NOT NULL,"
+                              + "  AlbumTitle   STRING(MAX)"
+                              + ") PRIMARY KEY (SingerId, AlbumId),"
+                              + "  INTERLEAVE IN PARENT Singers ON DELETE CASCADE",
+                          "ALTER DATABASE " + "`" + databaseId + "`"
+                              + " SET OPTIONS ( default_leader = '" + defaultLeader + "' )"))
+                  .build());
+      Database createdDatabase = op1.get();
```

Comment:
nit: You could try and call `.get()` after the `createDatabaseAsync()` to get the Database back and not expose the OperationFuture. We have some users that use the OperationMetadata directly, but the samples probably don't need it and can just expose only the Database.

i.e `Database createdDatabase = databaseAdminClient.createDatabaseAsync(request).get()`

Reply:
Sure, good suggestion. I have updated.